price_parser/parser.py

```diff
@@ -201,11 +197,18 @@ def extract_price_text(price: str) -> Optional[str]:
     m = re.search(r"""
         (\d[\d\s.,]*)  # number, probably with thousand separators
         \s*?           # skip whitespace
+        ([m|M,b|B]il\w*)?  # check million* or billion* 
```

Comment:
Please, [check out how [] works](https://docs.python.org/3/library/re.html#index-9).

Reply:
Thanks @Gallaecio for flagging it!
We can make it: `([m|M,b|B]il\w*)?`